src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/ConstantExpectedAttribute.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    /// <summary>
+    /// Indicates that the specified method parameter expects a constant
+    /// </summary>
+    /// <remarks>
+    /// This can be used to inform tooling that a constant should be used as an argument for the annotated parameter
+    /// </remarks>
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false, Inherited = false)]
+    public sealed class ConstantExpectedAttribute : Attribute
+    {
+        /// <summary>
+        /// Indicates the minimum inclusive bound of the constant expected
```

Comment:
NIT does this sound better?
```suggestion
        /// Indicates the minimum bound of the expected constant, inclusive.
```
Also seems all doc sentences need a period at the end.